ui/page/root/trade_page.go

```diff
@@ -117,65 +117,84 @@ func (pg *TradePage) layoutMobile(gtx C) D {
 }
 
 func (pg *TradePage) pageContentLayout(gtx C) D {
-	pg.dcrdexBtn.SetEnabled(true, &gtx)
-
-	pageContent := []func(gtx C) D{
-		pg.sectionTitle(values.String(values.StrExchangeIntro)),
-		pg.layoutAddMoreRowSection(pg.exchangeBtn, values.String(values.StrExchange), pg.Theme.Icons.AddExchange.Layout16dp),
-		pg.layoutAddMoreRowSection(pg.dcrdexBtn, values.String(values.StrDcrDex), pg.Theme.Icons.DcrDex.Layout16dp),
-	}
-
-	return cryptomaterial.LinearLayout{
-		Width:     cryptomaterial.MatchParent,
-		Height:    cryptomaterial.MatchParent,
-		Direction: layout.Center,
-	}.Layout2(gtx, func(gtx C) D {
+	return components.UniformPadding(gtx, func(gtx C) D {
 		return cryptomaterial.LinearLayout{
-			Width:  gtx.Dp(values.MarginPadding550),
-			Height: cryptomaterial.MatchParent,
+			Width:     cryptomaterial.MatchParent,
+			Height:    cryptomaterial.MatchParent,
+			Direction: layout.Center,
 			Margin: layout.Inset{
 				Bottom: values.MarginPadding30,
 			},
 		}.Layout2(gtx, func(gtx C) D {
-			return pg.Theme.List(pg.scrollContainer).Layout(gtx, len(pageContent), func(gtx C, i int) D {
-				return layout.Inset{
-					Right:  values.MarginPadding48,
-					Bottom: values.MarginPadding4,
-				}.Layout(gtx, pageContent[i])
-			})
+			return cryptomaterial.LinearLayout{
+				Width:  cryptomaterial.WrapContent,
+				Height: cryptomaterial.MatchParent,
+				Margin: layout.Inset{
+					Bottom: values.MarginPadding30,
+				},
+			}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+						layout.Rigid(pg.titleContent(values.String(values.StrExchangeIntro))),
+						layout.Rigid(pg.titleContent(values.String(values.StrExchangeIntroPt2))),
```

Comment:
Use layout.Inset{Bottom: values.MarginPadding10}.Layout(gtx, ....) for the last string layout so we can have a space btw the box and the last text as it is in the designs. Also, consider removing `pg.titleContent` because you'll need to handle the texts a bit differently. The first text dont't need `layout.Inset{Bottom: values.MarginPadding0}`.

<img width="659" alt="Screenshot 2023-10-04 at 8 34 42 PM" src="https://github.com/crypto-power/cryptopower/assets/57448127/695c662d-e26d-4ff8-98a6-bc5f4730649c">